proto/storage/compliance_integration.proto

```diff
@@ -13,6 +13,7 @@ message ComplianceIntegration {
   string version = 2 [(gogoproto.moretags) = 'search:"Compliance Operator Version,hidden,store"'];
   string cluster_id = 3 [(gogoproto.moretags) = 'search:"Cluster ID,hidden,store" sql:"fk(Cluster:id),no-fk-constraint,type(uuid),index=category:unique;name:compliance_unique_indicator"'];
   string compliance_namespace = 4;
-  // Collection of errors that occurred while trying to obtain collector health info.
+  // Collection of errors that occurred while trying to obtain compliance health info.
   repeated string status_errors = 5;
+  bool read_only = 6;
```

Comment:
nit:  Update `Next Tag`

Reply:
Fixed.
What is the reason to maintain a `Next Tag` field? I saw it is outdated on another proto message.

Reply:
That convention pre-dates me.  I think the idea is to make it easier for the next person when they add a field.  TBH, I have went back and forth as to whether it is useful or not.